tools/cvm-image-rewriter/README.md

```diff
@@ -37,7 +37,7 @@ framework, and the whole flow was divided into three stages:
 
 ### 2.1 Existing Plugins
```

Comment:
There is a 97-test directory.  Would it be possible to have a "sample" plugin setup directory and section 2.2 point to that?